lib/runtime/wasmer/exports_test.go

```diff
@@ -408,6 +408,32 @@ func TestInstance_GrandpaAuthorities_PolkadotRuntime(t *testing.T) {
 	require.Equal(t, expected, auths)
 }
 
+func TestInstance_BabeGenerateKeyOwnershipProof_NodeRuntime(t *testing.T) {
+	tt := trie.NewEmptyTrie()
+	rt := NewTestInstanceWithTrie(t, runtime.NODE_RUNTIME, tt)
+	authorityID := types.AuthorityID{}
+	slot := uint64(1)
+	_, err := rt.BabeGenerateKeyOwnershipProof(slot, authorityID)
```

Comment:
The returned value is always an empty byte slice, can't we somehow use another runtime to test with actual values?

Reply:
resolving for https://github.com/ChainSafe/gossamer/issues/3003